src/os/ui/server/abstractloadingserver.js

```diff
@@ -124,11 +125,13 @@ os.ui.server.AbstractLoadingServer.prototype.finish = function() {
  */
 os.ui.server.AbstractLoadingServer.prototype.formatIcons = function() {
   var icons = os.ui.server.AbstractLoadingServer.base(this, 'formatIcons');
+
   if (this.getError()) {
     var message = 'Server failed to load. See the log/alerts window for details.';
-    icons += '<i class="fa fa-warning text-warning" title="' + message + '"></i>';
+    icons += `<i class="fa fa-warning text-warning" title="${message}"></i>`;
```

Comment:
While we're updating this line, here's the [FA5 icon](https://fontawesome.com/v5.15/icons/exclamation-triangle?style=solid).